radio/src/gui/128x64/model_setup.cpp

```diff
@@ -921,6 +921,23 @@ void menuModelSetup(event_t event)
             }
           }
         }
+        else if (IS_MODULE_CROSSFIRE(moduleIdx)) {
+          lcdDrawTextAlignedLeft(y, INDENT "Baudrate");
+          lcdDrawNumber(MODEL_SETUP_2ND_COLUMN, y, CROSSFIRE_BAUDRATES[g_eeGeneral.mavbaud], attr|LEFT);
+          if (attr) {
+            g_eeGeneral.mavbaud = DIM(CROSSFIRE_BAUDRATES) - 1 - checkIncDecModel(event, DIM(CROSSFIRE_BAUDRATES) - 1 - g_eeGeneral.mavbaud, 0, DIM(CROSSFIRE_BAUDRATES) - 1);
```

Comment:
I don't think you should reuse mavbaudrate here. Module specific data should be in ModuleData. PXX and multi are already saving their extra settings there and I think crossfire should do the same.

Reply:
Very much not, this would (if the setting is 1) cause a control loss on Horus in case of emergency mode (that var is not backed up).